### PR TITLE
macOS Mojave compilation

### PR DESCRIPTION
**Pull request by pik694**
`eleveldb` from `hexpm` does not compile on macOS, I propose an update of this dependency so it compiles flawlessly on macOS Mojave.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -25,6 +25,7 @@ defmodule OMG.Umbrella.MixProject do
 
   defp deps do
     [
+      {:eleveldb, git: "https://github.com/basho/eleveldb.git", branch: "develop", manager: :make, override: true},
```
Comment (pdobacz) [inline mix.exs]: two comments here:
1/ give a comment why this is overriden :)
2/ can we override to sth less volatile (tag perhaps? a stable branch? - `develop` seems like we could easily go into a broken set of dependencies, if one accidentally upgrades `mix.lock`)

Comment (pik694) [reply mix.exs]: I'm afraid that I could only refer to the exact commit, as the fix on `eleveldb` side was applied no sooner than 20 days ago and there is none tag including it.

**Comment by InoMurko** [comment]
The way I got it to work on OSX is by sticking with older command line tools `Command Line Tools (macOS 10.13) for Xcode 9.4.1`

File: mix.exs (hunk 1 above)
Comment (pdobacz) [reply mix.exs]: hm, would there be a way to solve this mojave-specific problem with less possibility to backfire to non-mojave users? :thinking: (I just saw Ino's comment - maybe this?)

Comment (InoMurko) [reply mix.exs]: I second  @pdobacz comment. This might be a bit wild :)

**Comment by pnowosie** [comment]
> older command line tools `Command Line Tools (macOS 10.13) for Xcode 9.4.1`
Yeah @InoMurko we also did some ugly hacks #535

**Comment by pnowosie** [comment]
Closes #535

**Comment by pik694** [comment]
After discussion with @pdobacz, we decided that we will postpone merging this PR until `eleveldb` with fixes is released (tagged on GitHub). Until now, we will prepare a temporary patch, so any macOS user can apply it.

**Comment by achiurizo** [comment]
Closing this as stale( also since the discussion decided to wait on `eleveldb` updates with new releases. )

**Pull request closed without merging**